mobly/snippet/client_base.py

```diff
@@ -101,8 +101,8 @@ def initialize(self):
       2. starting the snippet server on the remote device.
       3. making a connection to the snippet server.
 
-    If error occurs at any stage, this function will abort the initialization
-    process and call `stop` to clean up.
+    An error occurs at any stage will abort the initialization. Only errors in
+    the second and third stages will trigger `stop` to clean up.
```

Comment:
be clear about the stage names.
"second and third" is not specific enough

Reply:
Done.